homeassistant/components/group/__init__.py

```diff
@@ -609,7 +604,8 @@ def _set_tracked(self, entity_ids):
     @callback
     def _async_start(self, *_):
         """Start tracking members and write state."""
-        self._reset_tracked_state()
+        if self.tracking:
```

Comment:
Not sure if we need to check this, or it's fine to call `self._reset_tracked_state()`?

Reply:
This is the startup callback so it cannot have been tracking.